torch/csrc/jit/runtime/argument_spec.h

```diff
@@ -214,7 +213,9 @@ struct CompleteArgumentInfoPOD {
   unsigned defined : 1;
   unsigned requires_grad : 1;
   signed device : 14;
-  uint32_t total_dims; // all TensorInfoPODs are in CompleteArgumentSpec's
+  unsigned dev_type : 16;
+  unsigned
+      total_dims : 16; // all TensorInfoPODs are in CompleteArgumentSpec's
```

Comment:
This type is narrower than the olde type

Reply:
Is it large enough for holding the number of all the tensor operands' dimensions?
I will add overflow check when save the dimensions number to the CompleteArgumentInfoPOD::total_dims.